queue/src/fiskaltrust.Middleware.Localization.QueueIT/RequestCommands/PosReceiptCommand.cs

```diff
@@ -0,0 +1,106 @@
+        }
+        public override async Task<RequestCommandResponse> ExecuteAsync(ftQueue queue, ReceiptRequest request, ftQueueItem queueItem)
+        {
+            var receiptResponse = CreateReceiptResponse(queue, request, queueItem, CountryBaseState);
+
+            FiscalReceiptResponse response;
+            if (!request.IsVoid())
+            {
+                var fiscalReceiptinvoice = CreateInvoice(request);
+                response = await _client.FiscalReceiptInvoiceAsync(fiscalReceiptinvoice).ConfigureAwait(false);
+            }
+            else
+            {
+                var fiscalReceiptRefund = CreateRefund(request);
+                response = await _client.FiscalReceiptRefundAsync(fiscalReceiptRefund).ConfigureAwait(false);
+            }
+            receiptResponse.ftSignatures = _signatureItemFactoryIT.CreatePosReceiptSignatures(response);
+
+            if (!response.Success)
+            {
+                throw new Exception(response.ErrorInfo);
+            }
+
+            return new RequestCommandResponse
+            {
+                ReceiptResponse = receiptResponse,
+                ActionJournals = new List<ftActionJournal>()
+            };
+        }
+
+        private static FiscalReceiptInvoice CreateInvoice(ReceiptRequest request)
+        {
+            var fiscalReceiptRequest = new FiscalReceiptInvoice()
+            {
+                //TODO Barcode = "0123456789" 
+                //TODO DisplayText = "Message on customer display",
+                Items = request.cbChargeItems?.Select(p => new Item
+                {
+                    Description = p.Description,
+                    Quantity = p.Quantity,
+                    UnitPrice = p.UnitPrice ?? 0,
+                    Amount = p.Amount,
+                    VatGroup = p.GetVatGroup()
+                }).ToList(),
+                PaymentAdjustments = request.GetPaymentAdjustments(),
+                Payments = request.cbPayItems?.Select(p => new Payment
+                {
+                    Amount = p.Amount,
+                    Description = p.Description,
+                    PaymentType = p.GetPaymentType()
+                }).ToList()
+            };
+            return fiscalReceiptRequest;
+        }
+
+        private static FiscalReceiptRefund CreateRefund(ReceiptRequest request)
+        {
+            var fiscalReceiptRequest = new FiscalReceiptRefund()
+            {
+                //TODO Barcode = "0123456789" 
+                //TODO DisplayText = "Message on customer display",
+                Refunds = request.cbChargeItems?.Select(p => new Refund
+                {
+                    Description = p.Description,
+                    Quantity = Math.Abs(p.Quantity),
+                    UnitPrice = p.UnitPrice ?? 0,
+                    Amount = Math.Abs(p.Amount),
+                    VatGroup = p.GetVatGroup(),
+                    OperationType = p.GetRefundOperationType()
```

Comment:
This method can return `null`, is this okay here? If not, what should we use as the default value?